examples/parsl/map_reduce.py

```diff
@@ -12,6 +12,7 @@
 config = Config(
          executors=[RADICALExecutor(
                         label        = 'RADICALExecutor',
+                        bulk_mode    = False,
```

Comment:
Should we turn on the bulk mode for this example, or is there any drawback?

Reply:
@andre-merzky sure, that makes sense.

Reply:
Addressed.